.github/workflows/dockerimage-clang-10-ubuntu-mpich.yml

```diff
@@ -1,76 +0,0 @@
-###############################################################################
```

Comment:
I think you can remove `generate-workflow.pl`, `workflow-template.yml` and `workflows.ini`.